flavor/cook/utils.py

```diff
@@ -29,11 +32,19 @@ def WaitEvent(event: str, is_error: mp.Value = None):
 
     if event not in EventSet:
         raise ValueError("Unknown event {}".format(event))
-    while not os.path.exists(os.path.join(os.environ["OUTPUT_PATH"], event)):
+
+    output_path = os.environ.get("OUTPUT_PATH")
+    if not output_path:
+        raise EnvironmentError("OUTPUT_PATH environment variable is not set.")
+
+    while not os.path.exists(os.path.join(output_path, event)):
         time.sleep(1)
         if is_error.value != 0:
-            return
-    os.remove(os.path.join(os.environ["OUTPUT_PATH"], event))
+            return False
```

Comment:
I don't see the usage of the return bool value across the codebase. Perhaps `return` is enough.

Reply:
removed.